applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_FIC_element.hpp

```diff
@@ -226,6 +226,12 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) UPwSmallStrainFICElement
 
     ///----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 
+    /// Assignment operator.
+    UPwSmallStrainFICElement& operator=(UPwSmallStrainFICElement const& rOther);
+
+    /// Copy constructor.
+    UPwSmallStrainFICElement(UPwSmallStrainFICElement const& rOther);
+
```

Comment:
Hidden by private. I think the preferred alternative is explicitly deleting them.
This is brought to the attention by the re-ordering that clang format does here.
Out of scope, we will look at those things later.